src/fastga/models/performances/mission/mission_components/climb.py

```diff
@@ -119,74 +119,69 @@ def compute(self, inputs, outputs, discrete_inputs=None, discrete_outputs=None):
         mass_t = mtow - (m_to + m_tk + m_ic)
         mass_fuel_t = 0.0
         previous_step = ()
+        self.flight_points = []
```

Comment:
Isn't this operation already done in the __init__ ?

Reply:
Yes, by default the list is initialized in the init() but it needs to be re-initialized after each loop. Otherwise the openMDAO component keeps the data in memory.

Reply:
Ok